README.md

```diff
@@ -30,13 +32,24 @@ Click on the "Use this template" button on the GitHub template repository page t
      ```bash
      cp .env.example .env
      ```
-   - Open the `.env` file.
+   - Open the new created `.env` file.
    - Fill in your GitHub username, repository name, and the personal access token you created:
      ```plaintext
      GITHUB_TOKEN=your_personal_access_token_here
      REPO_OWNER=your_github_username
      REPO_NAME=your_repository_name
      ```
+3. - **Check .gitignore**: Ensure that your `.env` file is listed in your `.gitignore` file to prevent it from being committed to your GitHub repository.
```

Comment:
```suggestion
3. - **Check `.gitignore`**: Ensure that your `.env` file is listed in your `.gitignore` file to prevent it from being committed to your GitHub repository.
```